05_syntax_lab/04.py

```diff
@@ -1,8 +1,16 @@
-"""
-Write a program that reads lines from the user
-until an empty line is inserted.
-After the user typed in an empty line,
-print all previously inserted lines in reverse
-order (from last to first)
-"""
-
+"""
+      _      _      _
+   __(.)< __(.)> __(.)=
+   \___)  \___)  \___)   
+
+Python course - Verint
+Week 1
+
+"""
+reverse=""
+while True:
+    line = raw_input()
+    if len(line) == 0: break
+    reverse = line + reverse
+
+print(reverse)
```

Comment:
Great solution (and +1 for the ducks)

print statement in python2 is usually written without the parenthesis (unlike python3 which forces parenthesis in print). The reason is printing multiple values separated by comma does not work as expected when using parenthesis